.expeditor/verify.pipeline.yml

```diff
@@ -6,19 +6,26 @@ expeditor:
 
 steps:
 
-- label: run-tests-ruby-2.6
+- label: lint-ruby-3.0
   command:
-    - /workdir/.expeditor/buildkite/verify.sh
+    - RAKE_TASK=lint /workdir/.expeditor/buildkite/verify.sh
   expeditor:
     executor:
       docker:
-        image: ruby:2.6-stretch
+        image: ruby:3.0
 
 - label: run-tests-ruby-2.7
   command:
-    - CI_ENABLE_COVERAGE=1 /workdir/.expeditor/buildkite/verify.sh
+    - RAKE_TASK=default /workdir/.expeditor/buildkite/verify.sh
```

Comment:
`CI_ENABLE_COVERAGE` is needed to upload to sonarqube .. please don't remove this